local-cli/install/install.js

```diff
@@ -10,7 +11,17 @@ log.heading = 'rnpm-install';
 function install(args, config) {
   const name = args[0];
 
-  var res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
+  const projectDir = config.getProjectRoots()[0];
+  const isYarnAvailable =
+    yarn.getYarnVersionIfAvailable() &&
+    yarn.isGlobalCliUsingYarn(projectDir);
+
+  var res;
+  if (isYarnAvailable) {
+    res = spawnSync('yarn', ['add', name], spawnOpts);
+  } else {
+    res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
```

Comment:
@kureev thanks for suggestions. I'm thinking of it as well but there're some cases might need to supported:

**1. simple add/remove command:** it can be achieved by `manager.add('some-package')`
```
yarn add some-package
npm install some-package --save
```

**2. add/remove multiple deps:**  we can use `manager.add('package1 package2')`
```
yarn add package1 package2 package3
npm install yarn add package1 package2 package3 --save
```

**3. with additional arguments**
```
yarn add some-package --exact
npm install some-package --save --save-exact

yarn add some-package --dev --exact
npm install package2 --save-dev --save-exact
```
For this case, Do we need make it into a separated function eg. `manager.addDev('some-package')` or allow `add` to takes extra args (eg. `add(' ', isDev)`

**4 - RARE CASE: Caller want to stick with `npm`**

like `react-native-cli/index` which allows user can init project with `--npm` flag and it's  happen only before react-native have been installed - it doesn't use `local-cli` during its process. btw I don't think we need to support `--npm` flag in `local-cli` right now.